firestore/integration_test.go

```diff
@@ -2355,6 +2361,136 @@ func TestIntegration_BulkWriter(t *testing.T) {
 	})
 }
 
+func TestIntegration_AggregationQueries(t *testing.T) {
```

Comment:
can we add some tests for transactions?

Reply:
Added support for running aggregation queries in transactions. Added tests.